### ISSUE TITLE
we need ShoopingCartId in AddItemViewModel.cs (OCC-305)

### ISSUE THREAD

**Issue by infofromca** [OrchardCMS/OrchardCore.Commerce#511]
### Is your feature request related to a problem? Please describe

break the changes

### Describe the solution you'd like

in future, you will need it 

### Describe alternatives you've considered

[Jira issue](https://lombiq.atlassian.net/browse/OCC-305)

**Comment by sarahelsaig** [comment]
I don't think you will need it. `AddItemViewModel` is only used in the `~/api/shoppingcart/{shoppingCartId?}` endpoint, where you pass in shopping cart ID as part of the route.
Adding a property to a type is also not a breaking change.
